app/javascript/src/lib/dayjs.js

```diff
@@ -0,0 +1,10 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+export const calculateCreatedAgo = date => {
+  dayjs.extend(relativeTime);
+  return dayjs(date).fromNow();
+};
+
+export const formatDatetoWeekdayTime = date =>
+  dayjs(date).format("dddd, hh:mmA");
```

Comment:
This doesn't belong here. Move this to `utils.js` within the closest scope.

The `lib/dayjs` concern should be to only to make modifications to the global instance of `dayjs`.